### ISSUE TITLE
Allow searching contacts based on Bounce/Do Not Contact (DNC)

### ISSUE THREAD

**Issue by ipso** [mautic/mautic#2782]
What type of report is this:

| Q  | A
| ---| ---
| Bug report? | 
| Feature request? | 
| Enhancement? |  Y

## Description:
Allow searching contacts based on Bounce/Unsubscribe/Do Not Contact settings.

In addition, I think it would be nice if each contact in the contact list showed if they were set DNC/Bounced so its easy to tell from a glance without having to go into each separate contact. Perhaps by just showing a red circle with line through it on each row for DNC, and a yellow circle with line through it for Bounced.

**Comment by zenzozen** [comment]
+1!

**Comment by npracht** [comment]
Hello, i think by doing a segment with bounce/unsubscribe filters, it answers to the need.
@escopecz you could close it :)

**Comment by escopecz** [comment]
Yes, Norman is right. But I leave this open since searching for DNC is a slightly different approach.

**Comment by snds** [comment]
Sorry guys but from a UX perspective this is not doable, even trying to filter at a segment level still does not allow you to export a do not contact list. In my use case I want to be able to export a DNC list so as to clean another list against that.

**Comment by drimlike** [comment]
> Hello, i think by doing a segment with bounce/unsubscribe filters, it answers to the need.
> @escopecz you could close it :)

Yes it works for bounce/unsubscribe, but how do you see/select/count/extract emails that have been put DNC manually? i.e. we had a list of emails to DNC from an external source that we wanted to DNC in Mautic, we processed those emails manually, but they now disappeared in the system and there is no way to retrieve the list.

**Comment by escopecz** [comment]
The feature is prepared. Just waiting for testers https://github.com/mautic/mautic/pull/7786
